tests/test_date.py

```diff
@@ -224,13 +392,12 @@ def test_should_return_today(self, date_string):
         param('Ieri'),
     ])
     def test_should_return_yesterday(self, date_string):
-        today = datetime.utcnow().replace(hour=0, minute=0, second=0, microsecond=0)
-        yesterday = today - relativedelta(days=1)
-        date_data = self.parser.get_date_data(date_string)
-        self.assertIsNotNone(date_data['date_obj'],
-                             "could not parse date from: %s" % date_string)
-        self.check_equal(yesterday.date(),
-                         date_data['date_obj'].date(), date_string)
+        self.given_today(datetime.utcnow().replace(hour=0, minute=0, second=0, microsecond=0))
+        self.given_date_string(date_string)
+        self.given_yesterday()
+        self.when_date_data()
+        self.then_should_return_yesterday_is_not_none()
```

Comment:
We should put in names not what the actual assert does, but what does it mean in context of current test.